components/docs-chef-io/content/automate/servicenow_integration_install.md

```diff
@@ -0,0 +1,240 @@
+
+The short description of the incident indicates the failed profile:
+
+![Scan Failed Profile Description](/images/automate/SNOW_Failed_Profile_Scan.png)
+
+The associated Chef InSpec scans are shown on the 'Chef InSpec scans' tab of the incident.
+
+Setting the value to `node` creates an incident for each failed node. All Chef InSpec scans failing for a node will be associated with the corresponding incident.
+
+The short description of the incident indicates the failed node:
+
+![Scan Failed Node Description](/images/automate/SNOW_Failed_Node_Scan.png)
+
+The associated Chef InSpec scans are shown on the 'Chef InSpec scans' tab of the incident.
+
+#### Property x_chef_incident.assigned_to
+
+The `x_chef_incident.assigned_to property` has no default value.
+It should be set to the ServiceNow user ID to assign incidents raised by the application to.
+If the user is in an assignment group, that group will also be assigned automatically to the incident.
+This property can be left blank and an assignment group used instead.
+
+#### Property x_chef_incident.assignment_group
+
+The `x_chef_incident.assignment_group` property has no default value. It can be set rather than an individual user in the `x_chef_incident.assigned_to` property to assign the incident to a group.
+
+#### Property x_chef_incident.impact
+
+The `x_chef_incident.impact` property is the value to set in an incident raised by the application. This should be set as either `1`, `2` or `3`.
+
+#### Property x_chef_incident.urgency
+
+The `x_chef_incident.urgency` property is the value to set in an incident raised by the application. This should be set as either `1`, `2` or `3`.
+
+#### Property x_chef_incident.retention_days
+
+The `x_chef_incident.retention_days` property is the number of days to retain Chef Infra Client run and Chef InSpec scan records in ServiceNow after the corresponding incident is closed or after the association with an incident is removed.
+The default is set to 30 days and Chef Infra Client run and Chef InSpec scan records will remain in ServiceNow and be deleted after 30 days when the following conditions are met:
+
+* The corresponding incident is closed
+* The corresponding incident is deleted
+* The incident has been removed from the Chef Infra Client run or Chef InSpec scan record by a user update
+
+#### Property x_chef_incident.logging.enabled
+
+The `x_chef_incident.logging.enabled` property allows application logging to be enabled. Once enabled, logs can be viewed from the Chef incidents > Logs menu and the System logs > Application logs menu by authorized users. The default value is `no` and logs are disabled by default.
+
+#### Property x_chef_incident.logging.verbosity
+
+The `x_chef_incident.logging.verbosity` property sets the logging level when logging is enabled. The default value is `error` and the following levels are available:
+
+* `debug`
+* `warn`
+* `info`
+* `error`
+
+### Configure Chef Automate
```

Comment:
Can we link this info from [settings->Notification](https://docs.chef.io/automate/notifications/), so we can reach this from both places.